pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java

```diff
@@ -533,7 +532,7 @@ public static boolean shouldAntiAffinityNamespaceUnload(
 
     /**
      * It filters out brokers which owns topic higher than configured threshold at
-     * {@link ServiceConfiguration.loadBalancerBrokerMaxTopics}. <br/>
+     * {@link: ServiceConfiguration#loadBalancerBrokerMaxTopics}. <br/>
```

Comment:
The link here is also broken.
<img width="626" alt="image" src="https://user-images.githubusercontent.com/16974619/160533210-cf0f4031-fa71-41a6-a13c-f443b41ef765.png">

Please help check other `@link` as well.

Reply:
This is because the loadBalancerBrokerMaxTopics variable is private and will not be accessible out of this class. I think in order to access, we might have to allow private fields in Javadoc (-private). 
For now, I have just added the variable with #

Reply:
that also work. I will remove the link